src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -50,12 +50,13 @@ public abstract partial class Constraint
     {
         internal Constraint() { }
         [System.ComponentModel.DefaultValueAttribute("")]
+        [System.Diagnostics.CodeAnalysis.AllowNull]
```

Comment:
There are still a bunch of nullable enable and nullable disables in this file.  I assume that's temporary?

Reply:
Yes, these are temporary and correspond to the disable/enable in the implementations. All of these are about unannotated dependencies, I'll go back and finish these when that's done.